docs/reference/errors-and-warnings/NU1702.md

```diff
@@ -0,0 +1,23 @@
+---
+title: NuGet Warning NU1702
+description: NU1702 warning code
+author: nkolev92
+ms.date: 07/25/2024
+ms.topic: reference
+f1_keywords: 
+  - "NU1702"
+---
+
+# NuGet Warning NU1702
+
+<pre>ProjectReference 'C:\Code\Project2.csproj' was resolved using '.NETFramework,Version=v4.7.2' instead of the project target framework '.NETCoreApp,Version=v5.0'.
+This project may not be fully compatible with your project.
+</pre>
+
+### Issue
+
+`AssetTargetFallback` was used to select assets from a package. The warning let users know that the assets may not be 100% compatible.
+
+### Solution
+
+Change the target framework to one of the projects to make them fully compatible.
```

Comment:
I don't think this suggestion is correct. 
However, I think the current version could be better. 

I'll push a change.

Reply:
Are you saying the TFM should match one of the project TFMs? If so, it's written as a possessive, so it can't just be "to one of the projects", it needs to show the possessive with the apostrophe. Disclaimer: I'm not an English major so I can be wrong :)